templates/resources.jsx

```diff
@@ -54,17 +54,15 @@ export default function Resources (props) {
     updateSelectedFilter(filter);
     updateSelectedId(id);
 
-    // let $items;
-    // if (filter === 'all') {
-    //   $items = $('.resources__item');
-    // } else {
-    //   $items = $('.resources__item.is-' + filter);
-    // }
-
-    // if ($items.length < 0) return;
-    // console.log('$items is:');
-    // console.log($items);
-    // a11y.focusFirst($items);
+    let $items;
+    if (filter === 'all') {
+      $items = $('.resources__item');
+    } else {
+      $items = $('.resources__item.is-' + filter);
+    }
+
+    if ($items.length < 0) return;
+    a11y.focusFirst($items);
```

Comment:
Because the state is updated React will re-render the component and so $items will be recreated. Therefore you'll need to move this call into a `useEffect` hook.